temporal-testing-junit4/build.gradle

```diff
@@ -0,0 +1,165 @@
+// Run 'gradle checkUpdates' to find out which dependencies have newer versions
+plugins {
+    id 'java-library'
+    id 'net.ltgt.errorprone' version '1.3.0'
+    id 'net.minecrell.licenser' version '0.4.1'
+    id 'com.palantir.git-version' version '0.12.3'
+    id 'maven-publish'
+    id 'signing'
+    id 'de.marcphilipp.nexus-publish' version '0.4.0'
+    id 'name.remal.check-updates' version '1.1.6'
+}
+
+apply plugin: 'maven-publish'
+apply plugin: 'de.marcphilipp.nexus-publish'
+apply plugin: 'java'
+
+if (hasProperty('signing.keyId')) {
+    apply plugin: 'signing'
+    signing {
+        sign configurations.archives
+    }
+}
+
+group = 'io.temporal'
+version = getVersionName()
+archivesBaseName = "temporal-testing-junit4"
+
+description = '''Temporal Workflow Java SDK testing, JUnit 4.x integration'''
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+    withJavadocJar()
+    withSourcesJar()
+}
+
+dependencies {
+    errorproneJavac('com.google.errorprone:javac:9+181-r4173-1')
+    errorprone('com.google.errorprone:error_prone_core:2.4.0')
+
+    api project(':temporal-testing')
+
+    implementation group: 'junit', name: 'junit', version: '4.13.1'
```

Comment:
I think it should be api level dependency:

api group: 'junit', name: 'junit', version: '4.13.1'

Reply:
You're right, I've changed the level to `api`.